plugins/modules/ipa_dnsrecord.py

```diff
@@ -35,12 +35,13 @@
   record_type:
     description:
     - The type of DNS record name.
-    - Currently, 'A', 'AAAA', 'A6', 'CNAME', 'DNAME', 'PTR', 'TXT', 'SRV' and 'MX' are supported.
+    - Currently, 'A', 'AAAA', 'A6', 'CNAME', 'DNAME', 'NS', 'PTR', 'TXT', 'SRV' and 'MX' are supported.
     - "'A6', 'CNAME', 'DNAME' and 'TXT' are added in version 2.5."
     - "'SRV' and 'MX' are added in version 2.8."
+    - "'NS' are added in version 8.2."
```

Comment:
```suggestion
    - "'NS' are added in community.general 8.2.0."
```
The other versions refer to Ansible (and these two lines should be removed, but not in this PR).